app/config/database.js

```diff
@@ -1,12 +1,13 @@
 var mongoose = require('mongoose');
+var logger = require('winston');
 
 function connect(connectionString) {
     mongoose.connect(connectionString);
 
     var db = mongoose.connection;
     db.on('error', console.error.bind(console, 'connection error'));
     db.once('open', function callbck() {
-        console.log('Mongoose connected at: ', connectionString);
+        logger.info('Mongoose connected at: ', connectionString);
```

Comment:
`logger.info` is writing a specific logging level, these are completely customizable.